eth_utils/encoding/__init__.py

```diff
@@ -1,22 +1,14 @@
 from __future__ import absolute_import
 
-import sys
-
 from eth_utils.string import (
     force_bytes,
 )
 
 
-if sys.version_info.major == 2:
-    from .py2 import (  # noqa: F401
-        int_to_big_endian,
-        big_endian_to_int as _big_endian_to_int,
-    )
-else:
-    from .py3 import (  # noqa: F401
-        int_to_big_endian,
-        big_endian_to_int as _big_endian_to_int,
-    )
+from .py3 import (  # noqa: F401
+    int_to_big_endian,
+    big_endian_to_int as _big_endian_to_int,
```

Comment:
Instead of importing these, the `encoding/py3.py` module should be merged with `encoding/__init__.py` and then `encoding/__init_.py` should be moved to `eth_utils/encoding.py` and the `eth_utils/encoding` directory should be removed.